src/qt/intro.cpp

```diff
@@ -349,3 +343,27 @@ QString Intro::getPathToCheck()
     mutex.unlock();
     return retval;
 }
+
+void Intro::setSizeWarningLabel()
+{
+    requiredSpace = m_blockchain_size;
+    QString storageRequiresMsg = QObject::tr("At least %1 GB of data will be stored in this directory, and it will grow over time.");
+    if (ui->prune->isChecked()) {
+        uint64_t prunedGBs = std::ceil(m_prune_target * 1024 * 1024 / GB_BYTES);
+        if (prunedGBs <= requiredSpace) {
+            requiredSpace = prunedGBs;
+            storageRequiresMsg = QObject::tr("Approximately %1 GB of data will be stored in this directory.");
+        }
+        Intro::ui->lblExplanation3->setVisible(true);
+    } else {
+        Intro::ui->lblExplanation3->setVisible(false);
+    }
+    requiredSpace += m_chain_state_size;
```

Comment:
Move to start of function, otherwise `storageRequiresMsg` will have the wrong amount.

Reply:
Nope, see L352-354

Reply:
Not sure what's to see there?
<img width="438" alt="Schermafbeelding 2019-10-22 om 18 22 17" src="https://user-images.githubusercontent.com/10217/67307520-ec989480-f4f8-11e9-969d-7ffcd3748b50.png">
Everywhere it says 3 GB it should say ~7 GB; 2 GB pruning + 3 GB chainstate (`m_assumed_chain_state_size`).

Reply:
This has never been a very precise number because 3 GB more or less didn't matter, but with pruning we should show the correct numbers.

Reply:
Oh now I understand what you mean.
Will work on that tomorrow

Reply:
@ryanofsky These functions are not existent in my branch.
Should I copy them?

Reply:
> @ryanofsky These functions are not existent in my branch.
> Should I copy them?

Yes, that's why I linked to my branch.